cli/src/main/java/com/devonfw/tools/ide/tool/ToolCommandlet.java

```diff
@@ -230,183 +178,16 @@ public boolean install(boolean silent) {
    * @return {@code true} if the tool was newly installed, {@code false} if the tool was already installed before and
    *         nothing has changed.
    */
-  protected boolean doInstall(boolean silent) {
-
-    VersionIdentifier configuredVersion = getConfiguredVersion();
-    // install configured version of our tool in the software repository if not already installed
-    ToolInstallation installation = installInRepo(configuredVersion);
-
-    // check if we already have this version installed (linked) locally in IDE_HOME/software
-    VersionIdentifier installedVersion = getInstalledVersion();
-    VersionIdentifier resolvedVersion = installation.resolvedVersion();
-    if (isInstalledVersion(resolvedVersion, installedVersion, silent)) {
-      return false;
-    }
-    // we need to link the version or update the link.
-    Path toolPath = getToolPath();
-    FileAccess fileAccess = this.context.getFileAccess();
-    if (Files.exists(toolPath)) {
-      fileAccess.backup(toolPath);
-    }
-    fileAccess.symlink(installation.linkDir(), toolPath);
-    this.context.getPath().setPath(this.tool, installation.binDir());
-    if (installedVersion == null) {
-      this.context.success("Successfully installed {} in version {}", this.tool, resolvedVersion);
-    } else {
-      this.context.success("Successfully installed {} in version {} replacing previous version {}", this.tool,
-          resolvedVersion, installedVersion);
-    }
-    postInstall();
-    return true;
-  }
+  protected abstract boolean doInstall(boolean silent);
 
   /**
-   * This method is called after the tool has been newly installed or updated to a new version. Override it to add
-   * custom post intallation logic.
+   * This method is called after the tool has been newly installed or updated to a new version.
```

Comment:
Did you remove `Override it to add custom post intallation logic.` on purpose or by accident?

Reply:
I just saw your explanation in the other comment...